salesapp.js

```diff
@@ -83,7 +84,8 @@ var alKai = new CookieStore('Alkai', 2, 16, 4.6);
 
 var storeArray = [pikePlace, seaTac, seaCenter, capHill, alKai];
 var formStoreArray = [];
-
+var hourlyTotalsStorage = [];
```

Comment:
Group globals like this up at the top of the file. It'll make them a lot easier to find.